modules/api/php/endpoints/candidate/visit/visit.class.inc

```diff
@@ -89,10 +91,11 @@ class Visit extends Endpoint implements \LORIS\Middleware\ETagCalculator
     public function handle(ServerRequestInterface $request) : ResponseInterface
     {
         $pathparts = $request->getAttribute('pathparts');
+        $version   = $request->getAttribute('LORIS-API-Version');
```

Comment:
I would prefer to have it done inside handleGET so that both handleGET and handlePUT have a similar signature.